### PR TITLE
parallel_tc_command

### PR DESCRIPTION
**Pull request by zhaozhiyuan0256**
parallel the tc commands, accelerate the delay update and the queue filter update, reduce the time to around 3 seconds.

### CODE REVIEW

File: system.py [hunk 1]
```diff
@@ -76,49 +79,69 @@ def __init__(
     def init_eth(self, eth_name):
         cmd01 = self.ip + ": tc qdisc add dev " + eth_name + " root handle 1: htb"
         cmd02 = (
-                self.ip
-                + ": tc class add dev "
-                + eth_name
-                + " parent 1: classid 1:1 htb rate 50mbit"
+            self.ip
+            + ": tc class add dev "
+            + eth_name
```
Comment (wujs0226) [inline system.py:84]: Does the code style modification need a separate branch? @hs-zhuwenjun

**Review by wujs0226 — CHANGES_REQUESTED** [review]

**Comment by zhaozhiyuan0256** [comment]
This is the automatic formatting of my vscode. This branch mainly performs
parallel operations on the tc command.

Jiasheng Wu ***@***.***> 于2024年9月9日周一 12:39写道：

File: system.py [hunk 2]
```diff
@@ -1019,8 +1062,13 @@ def run(self):
             for gs in self.gs_list:
                 self.node_dict.get(gs.name).set_delay_all()
 
-            self.renew_delay_update_all_route(t)
+            print("update queue delay:")
+            for k, v in CURRENT_ROUND_DELAY_UPDATE_CMD.items():
```
Comment (hs-zhuwenjun) [inline system.py:1066]: what meaning of 'k' and 'v'?  The name of value should be meaningful.

**Review by hs-zhuwenjun — CHANGES_REQUESTED** [review]

**Comment by zhaozhiyuan0256** [comment]
1. The host.json is modified because the machine is reboot to repair the
broken core network, and after reboot, the interface of kvm to ovs is
changed.
2. DEBUG set to False to make the code in running mode, but not DEBUG mode.
3. The commented out and useless code has been deleted in the new committed
branch.
4. The variable name while lterating over a dictionary has been changed in
the new committed branch.

Zhu, Wenjun ***@***.***> 于2024年9月9日周一 13:32写道：

**Comment by hs-zhuwenjun** [comment]
please solve this conflict and pull this merge request again.

File: system.py (hunk 1 above)
Comment (zhaozhiyuan0256) [reply system.py]: We should use a consistent formatting plugin to standardize the code style.

File: hosts.json [hunk 3]
```diff
@@ -24,7 +24,7 @@
         "host": "host1",
         "bridge": "br0",
         "nic_name": "enp1s0",
-        "if_port": 28,
```
[the remark this answers is not in the document]
Comment (zhaozhiyuan0256) [reply hosts.json:27]: The configuration has changed after the virtual machine was restarted.

File: hosts.json [hunk 4]
```diff
@@ -36,7 +36,7 @@
         "host": "host1",
         "bridge": "br0",
         "nic_name": "enp1s0",
-        "if_port": 26,
+        "if_port": 7,
```
[the remark this answers is not in the document]
Comment (zhaozhiyuan0256) [reply hosts.json:39]: The configuration has changed after the virtual machine was restarted.

File: system.py [hunk 5]
```diff
@@ -18,19 +18,22 @@
 
 FAST_ROUTE = True
 
-DEBUG = True
+DEBUG = False
```
[the remark this answers is not in the document]
Comment (zhaozhiyuan0256) [reply system.py:22]: To ensure that the code can run directly after being cloned to the local machine, it needs to be set to true.

File: system.py [hunk 6]
```diff
@@ -76,49 +79,69 @@ def __init__(
...
+            + "ms"
         )
         self.filter_exist[index] = True
         do_cmd(cmd)
 
     def modify_eth_queue_delay(self, eth_name, index, delay):  # ip-str delay-float
-        cmd = (
-                self.ip
-                + ": tc qdisc change dev "
+        # cmd = (
```
[the remark this answers is not in the document]
Comment (zhaozhiyuan0256) [reply system.py]: It has been deleted.

File: system.py (hunk 2 above)
Comment (zhaozhiyuan0256) [reply system.py]: It has been changed to cmd_ip and cmd_txt, which are more descriptive names.

**Pull request merged**